packages/labs/rollup-plugin-minify-html-literals/src/test/minify-html-literals.spec.ts

```diff
@@ -0,0 +1,597 @@
+import {expect} from 'chai';
+import MagicString, {SourceMapOptions} from 'magic-string';
+import {Options as HTMLOptions} from 'html-minifier';
+import {ParseLiteralsOptions, parseLiterals} from '../lib/parse-literals.js';
+import {Template, TemplatePart} from '../lib/models.js';
```

Comment:
Nit - can be skipped until future. Prefer bare module imports in tests.

Reply:
The reason why we want to use package imports in tests is so that the tests can import dev or prod builds of the source under test based on just a import condition change. Our Rollup builds don't copy the test files, so we use the same test source against different builds.